manifests/init.pp

```diff
@@ -294,6 +296,7 @@
   Optional[Stdlib::Absolutepath] $https_cert = $::pulp::params::https_cert,
   Optional[Stdlib::Absolutepath] $https_key = $::pulp::params::https_key,
   Optional[Stdlib::Absolutepath] $https_chain = $::pulp::params::https_chain,
+  Optional[Stdlib::Absolutepath] $https_ca_cert = $::pulp::params::https_ca_cert,
```

Comment:
Would it be simpler to default this to `ca_cert` and remove the need for `pick`? Then the option could just be "CA cert for apache" IMO that's simpler for the user.